services/collector/collector.yaml

```
@@ -17,7 +17,12 @@ receivers:
       cpu: {}
       load: {}
       disk: {}
-      filesystem: {}
+      filesystem:
+        include_devices:
+          devices:
+            - /dev/sda1
+            - /dev/sdb1
+          match_type: strict
```

after-thought, maybe this is better (untested):
```suggestion
          devices:
            - /dev/sd..
          match_type: regex
```

---

things I'm filtering out include:

```
/dev/loop0 - /snap/core20/1270
/dev/loop1 - /snap/snapd/14295
/dev/sda15 - /boot/efi
```